src/sentry/tasks/relay.py

```diff
@@ -284,8 +283,7 @@ def compute_configs(organization_id=None, project_id=None, public_key=None):
         for key in ProjectKey.objects.filter(project__in=projects):
             configs[key.public_key] = None
     elif project_id:
-        projects = [Project.objects.get(id=project_id)]
-        for key in ProjectKey.objects.filter(project__in=projects):
+        for key in ProjectKey.objects.filter(project_id=project_id):
```

Comment:
Apologies for all the noise in this change, perhaps that wasn't the best idea.  But this line is the actual fix.